.openpublishing.redirection.json

```diff
@@ -1,6 +1,41 @@
 { 
   "redirections": [ 
     {   
+        "source_path": "aspnetcore/client-side/cdn.md", 
+        "redirect_url": "/aspnet/core//client-side/", 
```

Comment:
I'm tasked with doing the same over here and using your PR as a guide ... a nice, fun little Monday morning PR. :grinning: